Directory.cpp

```diff
@@ -22,39 +22,36 @@ void Directory::ls()
     for (auto& d : dirlist){
         std::cout<<(d->getName());
         std::cout<<std::endl;
-
     }
     for (auto& f : filelist){
         std::cout<<(f->getName());
         std::cout<<std::endl;
-
     }
 }
 
 void Directory::mkdir(std::string name)
 {
-    Directory* d=this->getelement(name);
-    File* f=this->getfileelement(name);
+    Directory* d=this->getElement(name);
+    File* f=this->getFileElement(name);
     if(d == nullptr){
         if(f == nullptr){
             dirlist.push_back(new Directory(name));}
-    else std::cout<<"mkdir: nem lehet a következő könyvtárat létrehozni: "<<name<<": A fájl már létezik"<<std::endl;
-    }
+        else std::cout<<"mkdir: nem lehet a következő könyvtárat létrehozni: "<<name<<": A fájl már létezik"<<std::endl;}
     else std::cout<<"mkdir: nem lehet a következő könyvtárat létrehozni: "<<name<<": A könyvtár már létezik"<<std::endl;
 }
 
 void Directory::touch(std::string name)
 {
-    Directory* d=this->getelement(name);
-    File* f=this->getfileelement(name);
+    Directory* d=this->getElement(name);
+    File* f=this->getFileElement(name);
     if(f == nullptr){
         if(d == nullptr){
-        filelist.push_back(new File(name));}
-    else std::cout<<"touch: nem lehet a következő fájlt létrehozni: "<<name<<": A könyvtár már létezik"<<std::endl;}
+            filelist.push_back(new File(name));}
+        else std::cout<<"touch: nem lehet a következő fájlt létrehozni: "<<name<<": A könyvtár már létezik"<<std::endl;}
```

Comment:
Az indentáláshoz a kapcsos zárójel is hozzátartozik. A `}`-t nem a sor végére írjuk, ha több soros blokkról van szó, hanem az utolsó utasítás utáni sorba. (Hogy az `else` ez után új sorban van-e, vagy sem, vagy hogy a nyitó `{` új sorban van-e, az már ízlés / konvenció kérdése.) https://en.wikipedia.org/wiki/Indentation_style